blog/models.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+from blog import db, login_manager
+from flask_login import UserMixin
+
+@login_manager.user_loader
+def load_user(user_id):
+    return User.query.get(int(user_id))
+
+class User(db.Model, UserMixin):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(20), unique=True, nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(60), nullable=False)
+
+class BlogPost(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
consider refactoring models.py to break each class into it's own file. Not a blocker for now but would be good as a next pr :)